gamestonk_terminal/common/technical_analysis/overlap_view.py

```diff
@@ -0,0 +1,149 @@
+"""TA Overlap View"""
+__docformat__ = "numpy"
+
+import os
+from typing import List
+
+import matplotlib.pyplot as plt
+import mplfinance as mpf
+import pandas as pd
+from pandas.plotting import register_matplotlib_converters
+
+import gamestonk_terminal.feature_flags as gtff
+from gamestonk_terminal.common.technical_analysis import overlap_model
+from gamestonk_terminal.config_plot import PLOT_DPI
+from gamestonk_terminal.helper_funcs import export_data, plot_autoscale
+
+register_matplotlib_converters()
+
+
+def view_ma(
+    ma: str,
+    s_ticker: str,
+    s_interval: str,
+    df_stock: pd.DataFrame,
+    window_length: List[int],
+    export: str,
+):
+    """Plots EMA technical indicator
+
+    Parameters
+    ----------
+    ma: str
+        Type of moving average.  Either "EMA" "ZLMA" or "SMA"
+    s_ticker : str
+        Ticker
+    s_interval : str
+        Interval of data
+    df_stock : pd.DataFrame
+        Dataframe of prices
+    window_length : int
+        Length of EMA window
+    export : str
+        Format to export data
+    """
+    if s_interval == "1440min":
+        price_df = pd.DataFrame(
+            df_stock["Adj Close"].values, columns=["Price"], index=df_stock.index
+        )
+    else:
+        price_df = pd.DataFrame(
+            df_stock["Close"].values, columns=["Price"], index=df_stock.index
+        )
+
+    l_legend = [s_ticker]
+    for win in window_length:
+        if ma == "EMA":
+            df_ta = overlap_model.ema(s_interval, df_stock, win)
+            price_df = price_df.join(df_ta)
+            l_legend.append(f"EMA {win}")
+        elif ma == "SMA":
+            df_ta = overlap_model.sma(s_interval, df_stock, win)
+            price_df = price_df.join(df_ta)
+            l_legend.append(f"SMA {win}")
+        elif ma == "ZLMA":
+            df_ta = overlap_model.zlma(s_interval, df_stock, win)
+            price_df = price_df.join(df_ta)
+            l_legend.append(f"ZLMA {win}")
+
+    if export:
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            f"{ma}{window_length}",
+            price_df,
+        )
+    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
+    ax.set_title(f"{window_length} Day {ma.upper()} on {s_ticker}")
```

Comment:
![image](https://user-images.githubusercontent.com/25267873/129806671-1b1a454e-5062-4a2e-b557-4fc8b615eba5.png)

Can we not have length on the title? The legend already has that information, and it looks weird being in brackets in the title

Reply:
On that one, can we also add `plt.xlim([df.index[0], df.index[-1]])? I hate to see that default extra margin ahah